docs/state.md

```diff
@@ -384,14 +384,38 @@ WindowStore
   ledgerInfo: {
     creating: boolean,
     created: boolean,
-    reconcileStamp: number,
-    reconcileDelay: ?,
-    delayStamp: ?,
-    transactions: Array,
+    delayStamp: timestamp,
```

Comment:
please use a js builtin type, not `timestamp`. see http://usejsdoc.org/tags-type.html